packaging/flatpak/com.ignite.Ignite.yml

```diff
@@ -0,0 +1,33 @@
+app-id: com.ignite.Ignite
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+command: ignite
+
+finish-args:
+  - --share=network
+  - --device=all
+  - --filesystem=host
+build-options:
+  append-path: /usr/lib/sdk/golang/bin
+  build-args:
+    - --share=network
+  env:
+    - GOBIN=/app/bin
+    - GOROOT=/usr/lib/sdk/golang
+
+modules:
+  - name: ignite
+    buildsystem: simple
+    build-commands:
+      - $GOROOT/bin/go build -mod=readonly -o ignt ./ignite/cmd/ignite
+      - install -Dm00755 ignt $FLATPAK_DEST/bin/ignite
+      - install -Dm00644 packaging/flatpak/logo.svg $FLATPAK_DEST/share/icons/hicolor/scalable/apps/ignite.svg
```

Comment:
because of those 4 lines, it would be great if this gets merged before tagging v28.